### PR TITLE
added security question

### PR DESCRIPTION
**Pull request by vivian-dai**
## Summary of changes

Added a question about security

## Checklist

- [X] I have read and followed the [contribution guidelines](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CONTRIBUTING.md).
- [X] I have read through the [Code of Conduct](https://github.com/freeCodeCamp/Developer_Quiz_Site/blob/main/CODE_OF_CONDUCT.md) and agree to abide by the rules.
- [X] This PR is for one of the available issues and is not a PR for an issue already assigned to someone else.
- [X] My PR title has a short descriptive name so the maintainers can get an idea of what the PR is about.
- [X] I have provided a summary of my changes.

closes #XXXXX

### CODE REVIEW

File: src/data/security-quiz.ts [hunk 1]
```diff
@@ -561,6 +561,16 @@ const securityQuiz = [
...
     Link: "https://www.freecodecamp.org/news/buffer-overflow-attacks/"
+  },
+  {
+    Question: "Which of the following operating systems are known as a hacking tool?",
+    Answer: "Kali Linux",
+    Distractor1: "Ubuntu",
+    Distractor2: "Windows",
+    Distractor3: "MacOS",
+    Explanation:
+      "Kali Linux is a Debian-based Linux distribution aimed at advanced Penetration Testing and Security Auditing.",
```
Comment (jdwilkin4) [inline src/data/security-quiz.ts:572]: the last bit doesn't need to be capitilized

```suggestion
      "Kali Linux is a Debian-based Linux distribution aimed at advanced penetration testing and security auditing.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by vivian-dai** [comment]
Hey @jdwilkin4 all requested changes have been committed :)

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**